src/ssl/support.cc

```diff
@@ -192,6 +193,36 @@ int Ssl::asn1timeToString(ASN1_TIME *tm, char *buf, int len)
     return write;
 }
 
+// Compare 2 IP addresses
+static int compare_ip_addresses(void *check_data, ASN1_OCTET_STRING *altname_ip) {
+    // Cast check_data to const char
+    const unsigned char* check_ip = reinterpret_cast<const unsigned char*>(check_data);
+    sockaddr_storage socket_info;
+    if (!Ssl::is_ip_address(check_ip, &socket_info)) {
```

Comment:
Please see if you can use Ip::Address::operator "=" instead of calling is_ip_address() here. If the assignment returns true, the resulting Ip::Address object should be usable for comparison.

Reply:
Updated commits now use the `=` for comparison.